impl/google/authentication/google_test.go

```diff
@@ -51,6 +51,8 @@ func TestGoogleValidateCreds(t *testing.T) {
 		t.Fatalf("Failed to create GoogleAuth: %v", err)
 	}
 
+	mux := runtime.NewServeMux()
```

Comment:
Why does this commit belong in this PR?  It looks unrelated (but I'm not familiar with runtime.AnnotateContext)

Reply:
This commit is in this PR because the function signature for `runtime.AnnotateContext` changed between grpc-gateway v1.2 and v1.3.

Reply:
OK - I didn't look too closely. :-)